test/homepage-noweb-test.js

```diff
@@ -41,24 +40,31 @@ suite.addBatch({
         },
         "and we visit the root URL": {
             topic: function() {
-                var browser;
-                browser = new Browser();
+                var cb=this.callback,
+                    browser = new Browser();
 
-                browser.visit("http://localhost:4815/", this.callback);
+                browser.visit("http://localhost:4815/", function(){
+                        cb(!browser.success, browser);
```

Comment:
More generally, why did you change this? It doesn't seem to me like this matches all of the test function signatures.